app/elements/doc/showcase.mjs

```diff
@@ -0,0 +1,39 @@
+export default function DocShowcase({ html, state }) {
+  const { attrs } = state
+  const { title, image, url, author = 'the Enhance team' } = attrs
+
+  return html`
+    <style>
+      :host {
+        --radius: 0.5em;
+      }
+
+      article {
+        background-color: var(--smoke-denim);
+        border-radius: var(--radius);
+        box-shadow: rgba(0, 0, 0, 0.1) 0px 1px 3px 0px,
+          rgba(0, 0, 0, 0.06) 0px 1px 2px 0px;
+      }
+
+      img {
+        border-radius: var(--radius);
+      }
+
+      a {
+        color: inherit;
+      }
+    </style>
+    <article class="mt0 mb2 p0 p2-lg">
+      <a href="${url}" target="_blank">
+        <figure>
+          <img src="${image}" alt="" class="mb0" />
+        </figure>
+        <h1 class="font-medium">${title}</h1>
+        <p class="mt0 leading3">
+          <slot></slot>
+        </p>
+        <p class="mt0"><span class="font-medium">Author:</span> ${author}</p>
```

Comment:
Can we wrap the author's name in a url so it can take them to their socials or homepage?

Reply:
Only issue with that is I'm wrapping the whole card in a link to make it all clickable. I could see about moving the author name out of the card to make it its own link tho!

Reply:
I am not married to the idea so no worries.

Reply:
It's a good idea, I'll take a look! :)